spec/spec_helper.rb

```diff
@@ -3,7 +3,6 @@
 unless RUBY_PLATFORM =~ /java/
   require 'curb'
   require 'patron'
-  require 'em-http'
```

Comment:
@c960657 why is it important to remove that `require`?

Reply:
On Ruby HEAD, even `require`'ing the library will raise because of the missing constant. Removing it here doesn't seem to affect the tests on the other Ruby versions AFAICT.

Reply:
@c960657 thank you for investigating that and explaining 👍